lib/pasarguard-backup.sh

```
@@ -272,78 +291,82 @@ send_backup_to_telegram() {
+        local escaped_time=$(escape_markdown_v2 "$backup_time")
+        local caption=$'📦 *Backup Information*\n'
+        caption+="🌐 *Server IP*: \`$escaped_server_ip\`"$'\n'
+        caption+="📁 *Backup File*: \`$escaped_filename\`"$'\n'
+        caption+="⏰ *Backup Time*: \`$escaped_time\`"$'\n\n'
+        caption+="$escaped_info_message"
+
+        for chat_id in "${chat_ids[@]}"; do
+            chat_id=$(echo "$chat_id" | xargs)
+            [ -z "$chat_id" ] && continue
+
+            local response=$(curl "${curl_proxy_args[@]}" -s -w "\n%{http_code}" -F chat_id="$chat_id" \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)